test/simpleTestForGenerator/ODC-By-1.0.txt

```diff
@@ -0,0 +1,195 @@
+# ODC Attribution License (ODC-By)
+
+### Preamble
+
+The Open Data Commons Attribution License is a license agreement intended to allow users to freely share, modify, and use this Database subject only to the attribution requirements set out in Section 4.
```

Comment:
This text seems to lack some line wrapping from [their canonical text/plain form][1].  To make it easier to verify that this is in fact their upstream text, can we generate this file programatically, with something like:

```console
$ curl -s https://opendatacommons.org/files/2018/02/odc_by_1.0_public_text.txt | dos2unix | sed 's/[[:space:]]*$//' >test/simpleTestForGenerator/ODC-By-1.0.txt
```

[1]: https://opendatacommons.org/files/2018/02/odc_by_1.0_public_text.txt

Reply:
I'm not sure I know what you mean - but line wrapping doesn't matter/need to match the canonical in any case.

Reply:
> I'm not sure I know what you mean...

You can probably get a similar effect by copy/pasting [their canonical text/plain form][1] from your browser.

> ... but line wrapping doesn't matter/need to match the canonical in any case.

Right, so why would we have line-wrapping preferences of our own, instead of just dropping in the text as provided by upstream (with the unix LF line endings for consistency with the rest of our samples and removing their trailing whitespace to make `git show --check` happy)?  By sticking with their wrapping, we make it easier for folks to check our text against text asserted by opendatacommons.org.  Assuming upstream doesn't drastically re-wrap, this also makes for less churn if/when we update to keep up with their changes (e.g. #600, #608).

[1]: https://opendatacommons.org/files/2018/02/odc_by_1.0_public_text.txt

Reply:
I believe the text .txt files have no wrapping, and I used that to then create the xml file (roughly, it is a bit onerous, the whole process) and then wrapped by hand where needed. considering white space does not matter for matching, I don't think this matters either. let's leave it be and focus on any substantive problems